scss/_settings_colors.scss

```diff
@@ -43,15 +43,14 @@ $colors--light-theme: (
   text-hover: #757575,
   text-disabled: #666,
   text-default: #111
-);
+) !default;
 
 //text-hover: minimum contrast on primary that satisfies contrast checker AA
 $colors--dark-theme: (
   background: hsl(0, 0%, 15%),
```

Comment:
Out of curiosity - why not changing background color (as it is already used by the nav)?
Do we want to keep `background` the same, because it's used elsewhere?